Documentation/building/unix-instructions.md

```diff
@@ -67,6 +67,18 @@ In addition to the above packages, the runtime versions of the packages listed
 in the native section should also be installed (this happens automatically on
 most systems when you install the development packages).
 
+### Windows Subsystem For Linux
```

Comment:
Looks good to me. I am wondering if you can make this generic to any Linux distro which is using "C" as a default locale?